pt_miniscreen/pages/root/projects/overview.py

```diff
@@ -0,0 +1,124 @@
+from pt_miniscreen.pages.root.projects.project import Project
+from pt_miniscreen.pages.root.projects.config import ProjectConfig
+from pt_miniscreen.pages.root.projects.project_page import ProjectPage
+from pt_miniscreen.pages.root.projects.utils import (
+    PACKAGE_DIRECTORY,
+    EmptyProjectRow,
+    Row,
+)
+from pt_miniscreen.utils import get_image_file_path
+
+
+from pt_miniscreen.pages.root.projects.utils import InvalidConfigFile
+
+logger = logging.getLogger(__name__)
+
+
+class LogsPage(ScrollableTextFile):
+    def __init__(self, project_config, **kwargs) -> None:
+        super().__init__(path=project_config.logfile, **kwargs)
+
+
+class OverviewProjectPage(EnterableSelectableList):
+    animate_enterable_operation = False
+
+    def __init__(
+        self, project_config: ProjectConfig, on_delete: Callable, **kwargs
+    ) -> None:
+        rows: List[partial] = [
+            partial(
+                Row,
+                title="Run",
+                enterable_component=partial(ProjectPage, project_config),
+            ),
+            partial(
+                Row,
+                title="View Logs",
+                enterable_component=partial(LogsPage, project_config),
+            ),
+        ]
+
+        if PACKAGE_DIRECTORY not in project_config.path:
+
+            def remove_project():
+                Project(project_config).remove()
+                if callable(on_delete):
+                    on_delete()
+
+            rows.append(
+                partial(
+                    Row,
+                    title="Delete",
+                    enterable_component=partial(
+                        ConfirmationPage,
+                        title="Really delete?",
+                        confirm_text="Yes",
+                        cancel_text="No",
+                        on_confirm=remove_project,
+                        on_cancel=None,
+                    ),
+                )
+            )
+
+        super().__init__(
+            Rows=rows,
+            num_visible_rows=3,
+            **kwargs,
+        )
+
+    def bottom_gutter_icon(self):
+        if isinstance(self.selected_row, Row) and self.selected_row.text.text == "Run":
+            return get_image_file_path("gutter/play.png")
+        return super().bottom_gutter_icon()
+
+
+class ProjectOverviewList(EnterableSelectableList):
+    def __init__(self, directory, **kwargs) -> None:
+        self.directory = directory
+        super().__init__(Rows=self.load_rows(), **kwargs)
+
+    def load_rows(self) -> List:
+        def on_delete():
+            self.update_rows(rows=self.load_rows())
+
+        rows: List[Union[Type[EmptyProjectRow], partial[Row]]] = []
+
+        files = Path(self.directory).glob("*/project.cfg")
+
+        # Sort found files by date/time of last modification
+        for file in sorted(files, key=os.path.getmtime, reverse=True):
+            try:
+                logger.info(f"Trying to read {file}")
```

Comment:
being picky but these should probably be debug logs - this could be logged dozens of times each time this list is loaded, which isn't even really a user action. worth keeping in mind anyway